src/components/tree/phyloTree/helpers.ts

```diff
@@ -156,15 +170,19 @@ export const formatDivergence = (divergence) => {
  * This differs depending on which tree is in view so it's helpful to access it
  * by reaching into phyotree to get it
  */
-export const getIdxOfInViewRootNode = (node) => {
+export const getIdxOfInViewRootNode = (node: ReduxNode): number => {
   return node.shell.that.zoomNode.n.arrayIdx;
 };
 
 /**
  * Are the provided nodes within some divergence / time of each other?
  * NOTE: `otherNode` is always closer to the root in the tree than `node`
  */
-function isWithinBranchTolerance(node, otherNode, distanceMeasure) {
+function isWithinBranchTolerance(
+  node: ReduxNode,
+  otherNode: ReduxNode,
+  distanceMeasure: Distance,
+) {
```

Comment:
[not a request for changes]

Here the return type is not specified, but tsc can infer it (`boolean`). The function below, `findFirstBranchWithAMutation`, specifies the return type (`ReduxNode`) but we can drop this and tsc will infer it. Any guidance on what style to follow?